### PR TITLE
Added water heater doc and moved econet to water heater

### PR DESCRIPTION
**Pull request by w1ll1am23**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17322

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/water_heater.econet.markdown [hunk 1]
```diff
@@ -8,7 +8,7 @@ comments: false
 sharing: true
 footer: true
 logo: econet.png
-ha_category: Climate
+ha_category: Water heater
 ha_release: 0.61.0
 ha_iot_class: "Cloud Polling"
 ---
```
Comment (flowolf) [inline source/_components/water_heater.econet.markdown:15]: this needs a redirect header. see docs: [Renaming Pages](https://developers.home-assistant.io/docs/en/documentation_standards.html#renaming-pages)

**Review by flowolf — CHANGES_REQUESTED** [review]
rest LGTM. thanks!

File: source/_components/water_heater.markdown [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+layout: page
+title: "Water Heater"
+description: "Instructions on how to setup water heater devices within Home Assistant."
+date: 2018-10-10 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
```
Comment (frenck) [inline source/_components/water_heater.markdown:10]: missing logo, release, category...
see other components for examples.

File: source/_components/water_heater.markdown [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+```yaml
+# Example configuration.yaml entry
+water_heater:
+  platform: demo
+```
+
+## {% linkable_title Services %}
+
+### {% linkable_title Water heater control services %}
+Available services: `climate.set_temperature`, `climate.set_away_mode`, `climate.set_operation_mode`
```
Comment (frenck) [inline source/_components/water_heater.markdown]: These services changed, right? (`climate`)

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/water_heater.markdown (hunk 2 above)
Comment (w1ll1am23) [reply source/_components/water_heater.markdown:10]: You are looking at the base compoent, the econet water heater was copied from the econet climate. If you look at water_heater.econet.markdown it has the release, logo, and category.

File: source/_components/water_heater.markdown (hunk 3 above)
Comment (w1ll1am23) [reply source/_components/water_heater.markdown]: correct, i'll fix that. Forgot to update those.

File: source/_components/water_heater.econet.markdown (hunk 1 above)
Comment (w1ll1am23) [reply source/_components/water_heater.econet.markdown:15]: Thanks, I'll add that.

File: source/_components/water_heater.markdown (hunk 2 above)
Comment (frenck) [reply source/_components/water_heater.markdown:10]: 👍

**Review by frenck — DISMISSED** [review]

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**